pdf_parser/pdf_parse.py

```diff
@@ -125,10 +28,11 @@ def parse_pdf_document_pdftxt(document):
         with open(os.devnull, 'w') as FNULL:
             subprocess.check_call(cmd, stdout=FNULL)
     except subprocess.CalledProcessError as e:
+        err = e.sdterr if e.stderr else ''
         logger.warning(
             "The pdf [%s] could not be converted: %s",
             document.name,
-            e.sdterr,
```

Comment:
What is `e.stderr` if it's not truth-y? Is there a difference between an empty string and None?

Perhaps changing the logging token to be `%r` (repr, aka unambiguous representation of an object) would be simpler and make for better debugging?

Reply:
Maybe `%r` would work. I though e.stderr was the best solution, but it seems that sometimes pdftohtml crashs without sending a stderr for no apparent reasons, and it leads to an error when I try to display it.
Will try with `%r` 👍

Reply:
plus don't you have this information in a variable above? `err`